pkg/server/plugin/bootstrapper/k8sbootstrapper/k8sbootstrapper.go

```diff
@@ -0,0 +1,194 @@
+package k8sbootstrapper
+
+import (
+	"bytes"
+	"context"
+	"encoding/pem"
+	"net/http"
+	"sync"
+
+	"github.com/hashicorp/hcl"
+	spi "github.com/spiffe/spire/proto/common/plugin"
+	"github.com/spiffe/spire/proto/server/bootstrapper"
+	"github.com/zeebo/errs"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	bootstrapErr = errs.Class("k8s-bootstrapper")
+)
+
+const (
+	defaultNamespace    = "spire"
+	defaultConfigMap    = "spire-bootstrap"
+	defaultConfigMapKey = "bootstrap.crt"
+)
+
+type ConfigUpdaterConfig struct {
```

Comment:
Does this need to be exported?

Reply:
Nope. Other plugins also have exported config for no good reason :)